packages/templates/src/users/Form.tsx

```diff
@@ -1,19 +1,23 @@
+import { Signer } from '@ethersproject/abstract-signer'
 import { useToast } from '@chakra-ui/react'
 import { Account, UserFormEdit } from '@nft/components'
-import { useSession } from '@nft/hooks'
 import useTranslation from 'next-translate/useTranslation'
 import { useRouter } from 'next/router'
 import React, { useCallback, VFC } from 'react'
 import { useGetAccountQuery } from '../graphql'
 import useLoginRedirect from '../hooks/useLoginRedirect'
+import { useWeb3React } from '@web3-react/core'
 
 export const Template: VFC<{
   uploadUrl: string
-}> = ({ uploadUrl }) => {
+  signer: Signer | undefined
+  ready: boolean
+}> = ({ uploadUrl, signer, ready }) => {
   const { t } = useTranslation('templates')
   const { push } = useRouter()
-  const { account, signer } = useSession()
-  useLoginRedirect()
+  const { account } = useWeb3React()
+  useLoginRedirect(ready)
+
   const toast = useToast()
 
   const { data } = useGetAccountQuery({
```

Comment:
The `address` variable for this query needs to be converted to lowercase. My Edit Profile page isn't working because of this.

Which makes me wonder - when should we be getting `account` from `useWeb3React` and when should we be getting the `currentAddress` from `LiteflowContext`?

Reply:
Good catch, I updated it and a few other places https://github.com/liteflow-labs/liteflow-js/pull/90/commits/764aa50d62dc68ba646de0c73abacd0efbc29732

Reply:
Actually all the fix I did here were already done in the next PR to move the template to the app https://github.com/liteflow-labs/marketplace-template/pull/45 💪